### PR TITLE
APP-93138 update sdk profiling section

### PR DESCRIPTION
**Pull request by noambartouv**
This change is [Reviewable](https://reviewable.io/reviews/pendo-io/pendo-mobile-sdk/100)

### CODE REVIEW

**Review by TehilaTaub — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pendo-io/pendo-mobile-sdk/100#-:-Nm6sIod-LBI6Nm0S9ep:bnfp4nl)* status: 0 of 8 files reviewed, all discussions resolved

File: README.md [hunk 1]
```diff
@@ -57,13 +57,41 @@ Visit the [Mobile Resource Center](https://support.pendo.io/hc/en-us/categories/
...
+
+Network requests are executed in parallel, prioritizing guides with higher priority to be downloaded first and becoming available earlier during the initialization process of the session. For example, guides with an app launch trigger will be available immediately, while lower-priority guides may take a few additional seconds to be available during the session.
+
+**Outgoing traffic**
+
+The analytics captured by the SDK include starting a session, the application state (foreground and background), screen changing, clicks, guide events, and manual track events. The size of each analytic event varies from 1KB to 6.5KB.
+
+**Network usage example**
+
+A session with a single guide including four steps, 6 session analytics, 42 page transitions 65 feature clicks, 13 guide events
```
Comment (shlomipendo) [inline README.md:93]: missing comma between "42 page transitions" and "65 feature clicks"

**Review by shlomipendo — APPROVED** [review]

**Review by randmaayan — APPROVED** [review]
> Reviewed 7 of 8 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pendo-io/pendo-mobile-sdk/100)* status: all files reviewed, 1 unresolved discussion (waiting on @noambartouv)

**Comment by randmaayan** [comment]
[merge]

**Comment by pendo-ops** [comment]
All JIRA issue transitioned/labeled.
* [APP-93138](https://pendo-io.atlassian.net/browse/APP-93138): Jira Issue APP-93138: Changed to Resolved


Branch APP-93138-update-sdk-profiling deleted

**Pull request merged**